builtInFunctions/esdtNFTCreate.go

```diff
@@ -233,6 +238,26 @@ func (e *esdtNFTCreate) ProcessBuiltinFunction(
 	return vmOutput, nil
 }
 
+func (e *esdtNFTCreate) getTokenType(tokenID []byte) (uint32, error) {
+	if !e.enableEpochsHandler.IsDynamicESDTEnabled() {
+		return uint32(core.NonFungible), nil
+	}
+
+	systemAcc, err := e.getAccount(vmcommon.SystemAccountAddress)
+	if err != nil {
+		return 0, err
+	}
+
+	esdtTokenKey := append([]byte(baseESDTKeyPrefix), tokenID...)
```

Comment:
I think we already have a function of getGlobalMetaData , use that one.

Reply:
Done.